dimagi/pages/templates/sections/commcare/product_onboarding/onboard/_base.html

```diff
@@ -0,0 +1,39 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+<div class="column">
+  <a class="tile feature-projects pad-vert-line pad-horz-line border-rounded-large {% block feature_class %}{% endblock %}">
+    <div class="align-center pad-vert-small {% block icon_class %}{% endblock %}">
+      {% block icon %}{% endblock %}
+    </div>
+
+    <h5 class="align-center uppercase font-weight-medium {% block title_class %}{% endblock %}">
```

Comment:
i probably should have noticed this when I copied everything over, but semantically this tag should be a `<h3>` (keep the current styling though) and the pricing element should be a `<div>` not a header tag... would you be able to update this?

Reply:
Updated, but for the pricing element <div> is looking kinda small.
![onboarding](https://user-images.githubusercontent.com/28800222/79095923-5fcce600-7d79-11ea-9c67-a85cf087effd.png)

Reply:
@hemanthReddy06 you can apply the h5 class to the element to style it as an h5 but not semantically treat it as an h5 (two different things)

Reply:
@biyeun Updated can you take a look.